checker/jdk/nullness/src/java/util/AbstractMap.java

```diff
@@ -153,7 +153,7 @@ public boolean containsValue(@Nullable Object value) {
      */
     @Pure
     @EnsuresKeyForIf(result=true, expression="#1", map="this")
-    public boolean containsKey(@Nullable Object key) {
+    public boolean containsKey(Object key) {
```

Comment:
Other methods in this class that would benefit from `@Nullable Object` parameters: `containsValue`, `get`, `remove`.

Reply:
`AbstractMap` specifies that a NullPointerException is possible from all of those methods.  :-(